cmd/snaplock/runinhibit/inhibit.go

```diff
@@ -116,11 +120,13 @@ func removeInhibitInfoFiles(snapName string) error {
 	return nil
 }
 
-// LockWithHint sets a persistent "snap run" inhibition lock, for the given snap, with a given hint.
+// LockWithHint sets a persistent "snap run" inhibition lock, for the given snap, with a given hint
+// and saves given info that will be needed by "snap run" during inhibition (e.g. snap revision).
 //
 // The hint cannot be empty. It should be one of the Hint constants defined in
 // this package. With the hint in place "snap run" will not allow the snap to
-// start and will block, presenting a user interface if possible.
+// start and will block, presenting a user interface if possible. Also
+// info.Revision must be provided and cannot be unset.
```

Comment:
Also info.Revision corresponding to the snap revision that was installed must be provided ...

or something along those lines

Reply:
updated, thank you!